gslib/commands/cp.py

```diff
@@ -282,82 +276,79 @@
     Copying file://obj [Content-Type=text/plain]...
     Uploading   gs://your-bucket/obj:                                182 b/182 B
 
-    If the checksum didn't match the service would instead reject the upload and
-    gsutil would print a message like:
+    If the checksums don't match, the service rejects the upload and
+    gsutil prints the following message:
```

Comment:
Recommend keeping "like" or "similar to" since the message the user receives will not be identical to the one we give here